docs/notes/2025.11.10.11.23.44.md

```
@@ -0,0 +1,311 @@
+const allText = (await Promise.all(allDocs.map(p => fs.readFile(p, "utf8")))).join("\\n");
+
+const dataFiles = await globs(["data/**/*", "!data/**/.gitkeep", "!data/**/*.md"]);
+const orphans = dataFiles.filter(p => !allText.includes(path.basename(p)));
+
+console.log(`# Orphans (${orphans.length})`);
+for (const p of orphans) console.log(p);
+
+// --- tiny glob helper ---
+async function globs(patterns) {
+  const { glob } = await import("glob");
+  const files = new Set();
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)